docs/en/user_guides/config.md

```diff
@@ -1 +1,709 @@
+  </tr>
+  <tr>
+    <td class="tg-9wq8">default_runtime.py<br></td>
+    <td class="tg-0pky">-</td>
+    <td class="tg-0pky"><a href="#env_config">Environment Configuration</a><br><a href="#hook_config">Hook Configuration</a><br><a href="#log_config">Log Configuration</a> <br><a href="#weight_config">Weight Loading Configuration</a> <br><a href="#eval_config">Evaluation Configuration</a> <br><a href="#vis_config">Visualizaiton Configuration</a></td>
+  </tr>
+  <tr>
+    <td class="tg-lboi" rowspan="2">dbnet</td>
+    <td class="tg-9wq8">_base_dbnet_resnet18_fpnc.py</td>
+    <td class="tg-0pky"><span style="font-weight:400;font-style:normal">-</span></td>
+    <td class="tg-0pky"><a href="#model_config">Network Configuration</a> <br><a href="#pipeline_config">Data Pipeline Configuration</a></td>
+  </tr>
+  <tr>
+    <td class="tg-9wq8">dbnet_resnet18_fpnc_1200e_icdar2015.py</td>
+    <td class="tg-0pky"><span style="font-weight:400;font-style:normal">-</span></td>
+    <td class="tg-0pky"><a href="#dataloader_config">Dataloader Configuration</a> <br><a href="#pipeline_config">Data Pipeline Configuration(Optional)</a></td>
+  </tr>
+</thead>
+</table>
+
+The final directory structure is as follows.
+
+```Python
+config
+├── textdet
+│   ├── _base_
+│   │   ├── datasets
+│   │   │   ├── icdar2015.py
+│   │   │   ├── icdar2017.py
+│   │   │   └── totaltext.py
+│   │   ├── schedules
+│   │   │   └── schedule_adam_600e.py
+│   │   └── default_runtime.py
+│   └── dbnet
+│       ├── _base_dbnet_resnet18_fpnc.py
+│       └── dbnet_resnet18_fpnc_1200e_icdar2015.py
+├── textrecog
+│   ├── _base_
+│   │   ├── datasets
+│   │   │   ├── icdar2015.py
+│   │   │   ├── icdar2017.py
+│   │   │   └── totaltext.py
+│   │   ├── schedules
+│   │   │   └── schedule_adam_base.py
+│   │   └── default_runtime.py
+│   └── crnn
+│       ├── _base_crnn_mini-vgg.py
+│       └── crnn_mini-vgg_5e_mj.py
+└── kie
+    ├── _base_
+    │   ├──datasets
+    │   └── default_runtime.py
+    └── sgdmr
+        └── sdmgr_novisual_60e_wildreceipt_openset.py
+```
+
+## Naming Rules
+
+MMOCR follows the following style for profile naming, and contributors to the code base need to follow the same naming rules. The file names are generally divided into four sections: algorithm information, module information, training information, and data information. Words that logically belong to different sections are connected by an underscore `'_'`, and multiple words in the same section are connected by a short horizontal line `'-'`.
+
+```Python
+{{algorithm info}}_{{module info}}_{{training info}}_{{data info}}.py
+```
+
+- algorithm info: the name of the algorithm, such as DBNet, CRNN, etc.
+
+- Module info: list some intermediate modules in the order of data flow, whose content depends on the algorithm task, and some modules strongly related to the model will be omitted to avoid overly long Config. The following examples are given.
```

Comment:
```suggestion
- module info: list some intermediate modules in the order of data flow. Its content depends on the algorithm, and some modules strongly related to the model will be omitted to avoid an overly long name. For example:
```